src/Backend/Core/Layout/Templates/FormLayout.html.twig

```diff
@@ -118,41 +129,59 @@
     <fieldset>
       <legend>{{ 'lbl.Titles'|trans|ucfirst }}</legend>
       <div class="checkboxTextFieldCombo">
-        <div class="form-group">
-          {{ form_widget(form.titleOverwrite) }}
-        </div>
-        <div class="form-group">
-          {{ form_label(form.title) }}
-          {{ form_widget(form.title) }}
-          {{ form_errors(form.title) }}
-          <small class="form-text text-muted">{{ 'msg.HelpPageTitle'|trans|raw }}</small>
-        </div>
+        <checkbox-enable-field
```

Comment:
seems like a lot of repeating config, can you create a macro for this so that it will become reusable?